app/src/main/java/org/jellyfin/androidtv/ui/browsing/StdGridFragment.java

```diff
@@ -514,10 +515,14 @@ public void onClick(View v) {
                 dismiss();
                 return null;
             });
-            alphaPicker.setGravity(Gravity.CENTER_HORIZONTAL);
+
+            HorizontalScrollView scrollView = new HorizontalScrollView(requireContext());
```

Comment:
I think it would make more sense to change the alphapicker to extend HorizontalScrollView, add the LinearLayout as a child and then add the buttons to the linearlayout.